### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by zelenins**
Added jest and supertest, created __tests__ folder for the test scripts in the root.
Run the test with npm test, it will run all scripts from __tests__ folder and all the scripts that have test in their name.
Created 3 unit tests for createAccount, getAccounts and getAccount functions.

### CODE REVIEW

**Review by niktcha — COMMENTED** [review]
Let's create ```test``` folder in ```src``` folder and put  ```__test__``` folder in there and rename it ```unit```.

File: .eslintrc.json [hunk 1]
```diff
@@ -13,10 +13,5 @@
         "ecmaVersion": 2018
     },
     "rules": {
-        "no-trailing-spaces": "error",
-        "no-empty": 1,
-        "no-console": 1,
-        "no-irregular-whitespace": 1,
-        "no-unused-vars":1
```
Comment (mjovanovic) [inline .eslintrc.json:20]: What are the reasons to remove all these rules?

**Review by niktcha — COMMENTED** [review]
tests are failing.

**Comment by zelenins** [comment]
> Let's create `test` folder in `src` folder and put `__test__` folder in there and rename it `unit`.

Done

> tests are failing.

Tests are fixed, there was a bad check on getAccounts test, should be good now

File: .eslintrc.json (hunk 1 above)
Comment (zelenins) [reply .eslintrc.json:20]: These are not the default rules, no other reason

**Review by mjovanovic — APPROVED** [review]

**Review by niktcha — APPROVED** [review]

**Pull request merged**